pvacseq/lib/net_chop.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import requests
+import csv
+import tempfile
+import re
+import os
+from time import sleep
+from lib.main import split_file
+
+cycle = ['|', '/', '-', '\\']
+methods = ['cterm', '20s']
+
+def main(args_input = sys.argv[1:]):
+    parser = argparse.ArgumentParser("pvacseq net_chop")
+    parser.add_argument(
+        'input',
```

Comment:
We've been naming these variables `input_file` and `output_file` in other parts of the program and should stay consistent.